pyproject.toml

```diff
@@ -166,6 +166,8 @@ example-app-with-view-override = {path = "examples/example_app_with_view_overrid
 factory-boy = "~3.3.1"
 # Factory Boy dependency. Don't upgrade to v26.x until breaking change is fixed https://github.com/joke2k/faker/issues/2070
 faker = ">=0.7.0,<26.0.0"
+# Library to access Kubernetes API used to develop K8s integration
+kubernetes = "31.0.0"
```

Comment:
Perhaps `^31.0.0`? It looks like this library is versioned in parallel with the k8s API version, i.e. this corresponds to API version 1.31, but it also looks like (from a quick read of the CHANGELOG.md) there have been breaking API changes in "major" releases of this library.

Reply:
@ubajze What's your experience with K8s and `kubernetes` Python library versioning?

Reply:
I never had any problems when there was a mismatch between the library and K8s API version. Also, as I understand, you will only use the "jobs" endpoint, which is pretty stable.

Another option is to use `requests`, but this adds some complexity, because you have to provide authentication method. I am not sure which method is used by default for kind.

Reply:
Let's go with `^31.0.0` for now then and let it be managed by Renovate as usual.

Reply:
Fixed